tensorflow/lite/micro/kernels/reduce.cc

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 #include "tensorflow/lite/c/builtin_op_data.h"
 #include "tensorflow/lite/c/common.h"
 #include "tensorflow/lite/kernels/internal/quantization_util.h"
+#include "tensorflow/lite/kernels/internal/reference/integer_ops/mean.h"
```

Comment:
`tensorflow/lite/micro/kernels/reduce.cc:21:10: fatal error: '/tensorflow/lite/kernels/internal/reference/integer_ops/mean.h' file not found
#include "tensorflow/lite/kernels/internal/reference/integer_ops/mean.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.`

Reply:
What command is run to reproduce this error?

Reply:
@patrikerdes here is the command https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/benchmarks/BUILD#L21

Reply:
This is still failing internally , can you please check

Reply:
Running `bazel build //tensorflow/lite/micro/benchmarks:conv_benchmark` locally I do not encounter the problem. Could you provide more detailed instructions for how to reproduce this error? @rthadur